disco_aws_automation/disco_deploy.py

```diff
@@ -559,6 +547,66 @@ def handle_blue_green_ami(self, pipeline_dict, ami, old_group,
             self._disco_elb.delete_elb(hostclass, testing=True)
         return False
 
+    def _create_scaling_schedule(self, pipeline_dict, group_name=None, hostclass=None):
```

Comment:
oh wait... you are calling `disco_aws.py`... i would suggest you break it out of `disco_aws.py`